composer.json

```diff
@@ -45,21 +44,29 @@
         "post-install-cmd": "./vendor/bin/run drupal:site-setup",
         "post-update-cmd": "./vendor/bin/run drupal:site-setup"
     },
-    "repositories": [
-        {
+    "repositories": {
+        "drupal": {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
+        },
+        "ec-europa/oe-poetry-client": {
+            "type": "git",
+            "url": "https://github.com/GilNovacomm/oe-poetry-client.git"
+        },
```

Comment:
@GilNovacomm  I've worked on the oe-poetry-client update, you can remove this repository here, so that we use dev-EWPP-917 on the main repository, directly.